hw2/05-form.js

```diff
@@ -1 +1,24 @@
 // Add your code here
+const form = document.querySelector('form');
+
+function handleSubmit(e) {
+  e.preventDefault();
+  const elementNames = ['full-name', 'email', 'registration-status', 'comments'];
+  const formElements = form.elements;
```

Comment:
If you create a group that all of these console logs would be a part of, it would make it a bit more clean in the Console.